core/modules/url_filter.h

```diff
@@ -21,6 +24,9 @@ class Flow {
   uint32_t dst_ip;
   uint16_t src_port;
   uint16_t dst_port;
+  uint32_t padding;
+
+  Flow() : padding(0) {}
```

Comment:
I'm not sure you should add this constructor -- padding should be getting value-initialized automatically if you don't have a constructor.